### PR TITLE
Projects page and integration with SemanticUI

### PR DESCRIPTION
**Pull request by Wondertan**
AragonUI seems to be not developed enough for our requirements. Unluckily, choosing it originally was a mistake. Instead, I am moving on [SemanticUI](https://react.semantic-ui.com/).

### CODE REVIEW

File: src/components/ProjectCard.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Card } from '@aragon/ui'
+
+export default ({ children }) => {
+    return (
+        <Card>
+            {children}
+        </Card>
+    )
```
Comment (Bidon15) [inline src/components/ProjectCard.js:9]: Do we still need Card from aragonUI? 
Maybe an alternative from semantic UI ?

File: src/routes/Projects.js [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+const Projects = () => {
+    return (
+        <Box>
+            <CardLayout columnWidthMin={columnWidthMin} rowHeight={rowHeight}>
+                <ProjectCard> HI </ProjectCard>
+                <ProjectCard> HI </ProjectCard>
+                <ProjectCard> HI </ProjectCard>
+            </CardLayout>
+        </Box>
```
Comment (Bidon15) [inline src/routes/Projects.js:17]: Same question. Full  migration to semantic ui from aragon or just partial? 🧐

File: src/components/ProjectCard.js (hunk 1 above)
Comment (Wondertan) [reply src/components/ProjectCard.js]: Outdated

File: src/routes/Projects.js (hunk 2 above)
Comment (Wondertan) [reply src/routes/Projects.js]: Outdated

**Comment by Wondertan** [comment]
So I decided to keep both UIs for some time, as it is really hard to rewrite `AccountModule` on Semantic. Also, both UIs seem to work properly without any issue for now, as long as the whole app is wrapped with Aragon's `Main` component.

**Review by Bidon15 — APPROVED** [review]

**Pull request merged**